lms/djangoapps/teams/toggles.py

```diff
@@ -0,0 +1,43 @@
+"""
+Togglable settings for Teams behavior
+"""
+from edx_toggles.toggles import SettingDictToggle
+from openedx.core.djangoapps.waffle_utils import CourseWaffleFlag
+
+
+# Course Waffle inherited from edx/edx-ora2
+WAFFLE_NAMESPACE = "openresponseassessment"
+TEAM_SUBMISSIONS_FLAG = "team_submissions"
+
+# .. toggle_name: FEATURES['ENABLE_ORA_TEAM_SUBMISSIONS']
```

Comment:
@regisb: FYI: Our toggle report (state + annotations) incorrectly had 2 rows for this.  One for the setting that was actually set name `FEATURES['ENABLE_ORA_TEAM_SUBMISSIONS']`, and one for the original annotation name `ENABLE_ORA_TEAM_SUBMISSIONS`.  I was wondering if we might want to warn on this situation, but then remembered that the linter can hopefully confirm that the toggle name matches the SettingDictToggle (or SettingToggle) definition. :)